src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -172,17 +174,6 @@ const StyledHeaderHelpText = styled(HelperText)`
   ${truncate}
 `;
 
-const StyledDropdownMenuContent = styled(radixDropdownMenu.Content)`
-  background-color: var(--echoes-color-background-default);
-  border: var(--echoes-border-width-default) solid var(--echoes-color-border-weak);
-  border-radius: var(--echoes-dimension-space-100);
-  box-shadow: var(--echoes-box-shadow-medium);
-  box-sizing: border-box;
-  margin: var(--echoes-dimension-space-50) 0;
-  max-height: var(--radix-dropdown-menu-content-available-height);
-  max-width: var(--echoes-sizes-overlays-max-width-default);
-  min-width: var(--echoes-sizes-overlays-min-width-default);
-  overflow-x: hidden;
-  overflow-y: auto;
-  padding: var(--echoes-dimension-space-50) var(--echoes-dimension-space-0);
-`;
+const StyledDropdownMenuContent = StyledDropdownMenuSubContent.withComponent(
+  radixDropdownMenu.Content,
+);
```

Comment:
I would have expected the reverse, or maybe even a common base that they both use (e.g. `StyledItem.withComponent`)

Reply:
yes, my concern was that DropdownMenu imports DropdownMenuSubMenu, so I wanted to avoid a circular dependency, and also avoid a new common base

Reply:
Extracting it to its own file is the way to go I think. We don't want the main component to depend on some "edge case" component styling.